### PR TITLE
[201807][platform_barefoot]Wedge100bf_32x, add device util, eeprom and psuutil.

### PR DESCRIPTION
**Pull request by roylee123**
These 2 utilities poll data through bfshell.

**- What I did**
Add eeprom.py and psuutil.py to device for wedge100bf_32x of barefoot.

**- How I did it**
It needs 2 more debain packages, python-pexpect and python-ptyprocess.
Use pexpect to grab data from bfshell.

**- How to verify it**
By "decode-syseeprom" and "psuutil status"

**- Description for the changelog**

### CODE REVIEW

File: device/barefoot/x86_64-accton_wedge100bf_32x-r0/plugins/eeprom.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        if now < (last_poll + self.CHECK_INTV):
+            return True
+        else:
+            return False
+
+    def interact_with_bfshell(self, cmds, timeout=20):
+        bfshell_path = 'docker exec -it syncd /opt/bfn/install/bin/bfshell'
+        bfshell_cmd = [ bfshell_path ]
+
+        prompt_list = ['bfshell> $', 'bf-sde\.*.*> $', 'pd-switch:\d+> $', pexpect.EOF]
```
Comment (lguohan) [inline device/barefoot/x86_64-accton_wedge100bf_32x-r0/plugins/eeprom.py]: do you you get eeprom/psu data from bfshell? Isn't your platform driver suppose create sysfs entries to expose those data?

Comment (lguohan) [reply device/barefoot/x86_64-accton_wedge100bf_32x-r0/plugins/eeprom.py]: ok. can you get this to master branch first? we need to ensure we have this in the master branch, then we can cherry-pick it back to a release branch.

Comment (roylee123) [reply device/barefoot/x86_64-accton_wedge100bf_32x-r0/plugins/eeprom.py]: Get it. 
I will make another PR for this change to master branch. But it needs quite a while for I'm working on other platforms.

**Comment by roylee123** [comment]
Postponed, there are code changes by other PR.

**Pull request closed without merging**